bin/varnishtest/tests/m00000.vtc

```diff
@@ -142,7 +142,7 @@ varnish v1 -errvcl {Symbol not found: 'obj'} {
 
```

Comment:
On the commit message:
As with other versioning questions, I think we should only increase the major version for _silent_ breaking changes. In this case a VCL compilation failure ensures that VCL authors become aware of the change, so we do not need a VCL version bump.

Reply:
In this case I think we should consider it a bug fix, we should never have authorized `none` to persist as a usable symbol name once we gave it a special meaning.

Reply:
Yes